test/testdata/deployednettemplates/recipes/mainnet-model/network-tpl.json

```diff
@@ -6,17 +6,17 @@
   "instances": {
     "relays": {
       "config": "./configs/relay.json",
-      "type": "c5.xlarge",
+      "type": "c5d.xlarge",
```

Comment:
Have you seen any measurable gains when switching the EBS -> NVMe ?
For the mainnet model, I do think that we shouldn't use EBS, but NVMe might be an overspec for non-relays. ( i.e. we should use a non-nvme SSD or spinning rust )

Reply:
Yes. This was crucial to getting the initial 1000 TPS benchmark. We should always be using local SSD for the tracker db.